modules/core/shared/src/main/scala/eu/timepit/refined/api/Min.scala

```diff
@@ -0,0 +1,51 @@
+package eu.timepit.refined.api
+
+trait Min[FTP] { def min: FTP }
```

Comment:
I'v used `FTP` in cases where the parameter is always a refined type. Since there are also instances for primitive types (`Int`, `Double`, etc.) I'd go with just `T` here. The same applies to `Max`.

Reply:
Good point - I'll fix